### PR TITLE
added login and register functions

### PR DESCRIPTION
**Pull request by andrew-zhao12**
Tests using these register and login
test0
test1
Result of using register function with new username and password
test2
test3
test4
test5
test6
Results on terminal with local testing

### CODE REVIEW

File: backend/functions/login.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    retrieve_users = connection.execute(retrieve_users).fetchall()
+    if retrieve_users[0][0] == 0:
+        return 1
+    retrieve_password = db.select([user_info]).where(user_info.columns.Username == username)
+    retrieve_password = connection.execute(retrieve_password).fetchall()
+    if retrieve_password[0][1] != password:
+        return 2
+    return 0
+
+print(login('andre','pissgan'))
```
Comment (jchen8460) [inline backend/functions/login.py:24]: I think this line can be removed as it seems to be for testing purposes.

**Review by jchen8460 — APPROVED** [review]
Looks good to me. Please remove the print statement.

**Review by magrwl — COMMENTED** [review]
This looks good.

File: backend/functions/login.py (hunk 1 above)
Comment (andrew-zhao12) [reply backend/functions/login.py:24]: will do

**Pull request merged**